### ISSUE TITLE
Accessibility features missing

### ISSUE THREAD

**Issue by zersiax** [Athena-OS/athena-iso#19]
Just gave this distro a whirl but it appears various accessibility features such as Orca, the screen reader that comes bundled with Gnome, are not present. This renders the entire product unusable for me unless I spend hours figuring out the exact things that were removed and compensate.
Can this be looked into so people like myself aren't excluded? Thanks much! :)

**Comment by D3vil0p3r** [comment]
Hey @zersiax I would like to discuss your proposal on our Discord server with the other guys. Can you reach there so you can detail better what you expect? Discord link: https://discord.gg/yammSxpA

Just be aware that we main consider to implement stuff on the purpose of Athena, so related to InfoSec. If you wish the resources you expect can be useful for InfoSec activity (also accessibility tools could be because they can improve productivity), most likely could be implemented.

Let's discuss it on Discord!

**Comment by D3vil0p3r** [comment]
The latest release has orca by default. The only remaining part is having accessibility on Calamares as we already discussed.

**Comment by D3vil0p3r** [comment]
I didn't forget this enhancement. We are migrating to a new installer in order to make it compliant with your need if we are able to do it. So, since there is no will on Calamares project to think about accessibility features, we abandoned it and we are creating a new installer trying to implement accessibility features.
